website/snippets/_install-dbt-extension.md

```diff
@@ -112,64 +112,121 @@ This will start the upgrade tool and guide you through the Fusion upgrade with a
 Once the upgrade is completed, you're ready to dive into all the features that the <Constant name="fusion_engine" /> has to offer!
 
 ## Register the extension
+After downloading the extension and installing the <Constant name="fusion_engine" />, you must register the dbt VS Code extension within 14 days of installing it (or re-installing it). 
+
+**Key points:**
```

Comment:
Could we call these "prerequisites" to be consistent with other areas of the docs?